algorand/admin.py

```diff
@@ -1333,6 +1333,7 @@ def testnet(self):
         self.tokenid = self.args.tokenid
 
     def mainnet(self):
+        #self.ALGOD_ADDRESS = "http://localhost:4001"
```

Comment:
Probably can remove

Reply:
Going to make this a settable argument for testing in another branch